### PR TITLE
[Fix] fix visualization for partition

### PR DESCRIPTION
**Pull request by AllentDan**
Introduced by #1280

### CODE REVIEW

File: mmdeploy/codebase/base/backend_model.py [hunk 1]
```diff
@@ -26,6 +27,9 @@ def __init__(self,
         if deploy_cfg is not None:
             ir_config = get_ir_config(deploy_cfg)
             output_names = ir_config.get('output_names', None)
+            if get_partition_config(deploy_cfg) is not None:
+                output_names = get_partition_config(
+                    deploy_cfg)['partition_cfg'][0]['output_names']
```
Comment (grimoire) [inline mmdeploy/codebase/base/backend_model.py:32]: what if the model will be partition into two part?

Comment (AllentDan) [reply mmdeploy/codebase/base/backend_model.py:32]: `output_names` should be passed into the class instead of `None`.

Comment (grimoire) [reply mmdeploy/codebase/base/backend_model.py:32]: two stage model will be partitioned into two part, which output_names will you accept?

**Review by grimoire — APPROVED** [review]

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**